internal/pkg/aws/ecs/task.go

```diff
@@ -123,6 +123,15 @@ func (t *TaskDefinition) EnvironmentVariables() map[string]string {
 	return envs
 }
 
+// Secrets returns secrets of the task definition.
+func (t *TaskDefinition) Secrets() map[string]string {
+	secrets := make(map[string]string)
+	for _, secret := range t.ContainerDefinitions[0].Secrets {
```

Comment:
why do we only display secret in the first container? main container might not always be the first one and we might want to display secrets in the sidecar containers as well. What do you think?

Reply:
Ah, okay. So range through ContainerDefinitions, then add any that aren't duplicates?

Reply:
i bet the env var suffers from the same problem. it requires more efforts because we allow them to have the same env vars. we need to display in a different format i think. maybe leave it as is for now since we don't support env var or secrets in sidecar https://github.com/aws/copilot-cli/issues/1119. We can address this problem once we resolve #1119.

Reply:
Haha--we didn't see this last comment so I went ahead and did it. :)